pkg/storage/stores/shipper/compactor/deletion/delete_request.go

```diff
@@ -3,22 +3,34 @@ package deletion
 import (
 	"github.com/prometheus/common/model"
 	"github.com/prometheus/prometheus/model/labels"
-	"github.com/prometheus/prometheus/promql/parser"
 
 	"github.com/grafana/loki/pkg/storage/stores/shipper/compactor/retention"
 )
 
-// DeleteRequest holds all the details about a delete request.
 type DeleteRequest struct {
 	RequestID string              `json:"request_id"`
 	StartTime model.Time          `json:"start_time"`
 	EndTime   model.Time          `json:"end_time"`
-	Selectors []string            `json:"selectors"`
+	Queries   []string            `json:"queries"`
```

Comment:
Not a strong opinion but `Selectors` still sound relevant here to me and less confusing than `Queriers`.

Reply:
Yeah, naming is hard. This was changed because we get more than a selector now. A line filter expression can also be present.

Reply:
`Query` sounds weird because it is for fetching the data but that could be just me. We can still call it `Selector` even with filter expression because it is selecting the logs to be deleted. Not strongly opposing it but as you said naming is hard :shrug: